CommandLine/emake/Makefile

```diff
@@ -1,35 +1,46 @@
-CXX=g++
-CXXFLAGS=-g -std=c++11 -c -Wall -I. -I../../CompilerSource
+BINARY := ../../emake
+SRC_DIR := .
+OBJ_DIR := .eobjs
 
 ifeq ($(OS),Windows_NT)
```

Comment:
This if doesn't actually work I have a fix for it in my mac fix pr. I never set $OS anywhere and OS isn't actually Windows_NT in mingw's shell anymore